optuna/study/study.py

```diff
@@ -1140,6 +1140,7 @@ def objective(trial):
 
     See also:
         :func:`optuna.create_study` is an alias of :func:`optuna.study.create_study`.
+        The :ref:`rdb` tutorial provides concrete examples to facilitate understanding.
```

Comment:
Nice!  How about this?
```suggestion

    See also:
        The :ref:`rdb` tutorial provides concrete examples to save and resume optimization using RDB.
```


1. The existing `see also` is not related to the new sentence, so could you separate them>
2. Explain a little bit the tutorial.

Reply:
I'm sorry, I'm not going to be able to finish in time....

Reply:
If you don't mind, I'll apply this suggestion :)